client/k8s.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
what's the intention of adding white space here?

Reply:
I use vim to format my code before commit-ing(Goland doesn't run goimports,gofmt, etc correctly each time) and it gets rid of the space when I save a file. I'll drop this change.